src/views/Beheer/Links/LinksBeheer.vue

```diff
@@ -24,14 +24,11 @@
     </li>
   </ul>
   <menu>
-    <router-link to="/Beheer/link/">
-      <utrecht-button
-        appearance="primary-action-button"
-        title="toevoegen"
-        type="button"
-        class="icon icon-after plus icon-only"
-      >
-      </utrecht-button>
+    <router-link
```

Comment:
wat is de richtlijn? gebruiken we de utrecht-button component helemaal niet meer?